### PR TITLE
Updated Zipkin spec for remoteEndpoint mapping.

### PR DESCRIPTION
**Pull request by CodeBlanch**
See open-telemetry/opentelemetry-dotnet#483

### CODE REVIEW

File: specification/exporter-zipkin.md [hunk 1]
```diff
@@ -64,6 +63,21 @@ Zipkin.
...
+|Priority|Attribute Name|Reason|
+|---|---|---|
+|1|net.peer.name|[OpenTelemetry adopted attribute for remote hostname, or similar.](./data-span-general.md#general-network-connection-attributes)|
+|1|peer.service|Remote service name defined in OpenTracing specification.|
+|2|peer.hostname|Remote hostname defined in OpenTracing specification.|
+|2|peer.address|Remote address defined in OpenTracing specification.|
+|3|http.host|Commonly used address attribute for Http Spans.|
+|3|db.instance|Commonly used address attribute for DB Spans.|
+
+Highest priority match should be selected.
```
Comment (SergeyKanzhelev) [inline specification/exporter-zipkin.md]: highest or lowest?

Comment (SergeyKanzhelev) [reply specification/exporter-zipkin.md]: what if priority match?

Comment (CodeBlanch) [reply specification/exporter-zipkin.md]: @SergeyKanzhelev Updated.

File: specification/exporter-zipkin.md [hunk 2]
```diff
@@ -64,6 +63,21 @@ Zipkin.
 | `SpanKind.PRODUCER`|`SpanKind.PRODUCER` ||
 |`SpanKind.INTERNAL`|`null` |must be omitted (set to `null`)|
 
+### Remote endpoint
+
+If Zipkin `SpanKind` resolves to either `SpanKind.CLIENT` or `SpanKind.PRODUCER` then the service SHOULD specify remote endpoint otherwise Zipkin won't treat the Span as a dependency. OpenTelemetry doesn't have a perfectly expressive field for the destination service, so the following mapping is applied:
+
+|Priority|Attribute Name|Reason|
+|---|---|---|
+|1|net.peer.name|[OpenTelemetry adopted attribute for remote hostname, or similar.](./data-span-general.md#general-network-connection-attributes)|
```
Comment (Oberon00) [inline specification/exporter-zipkin.md]: Should net.peer.ip + port also be considered?

Comment (CodeBlanch) [reply specification/exporter-zipkin.md]: @Oberon00 @SergeyKanzhelev Updated!

**Pull request closed without merging**